### PR TITLE
DDF-04481 Removed CAS protocol support through filters

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do?
Removes CAS protocol support through filters. Moving forward authentication with CAS will only be supported using SAML

#### Who is reviewing it? 
@SmithJosh @bakejeyner 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@brendan-hofmann
@garrettfreibott
@stustison 

#### How should this be tested?
Verify that DDF works the same. 

#### Any background context you want to provide?
Related to https://github.com/codice/ddf/pull/4620

#### What are the relevant tickets?
For GH Issues:
Fixes: https://github.com/codice/ddf/issues/4481

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/sdk/ddf-sso-widget/src/main/resources/OSGI-INF/blueprint/casfilters.xml [hunk 1]
```diff
@@ -1,92 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (blen-desta) [reply distribution/sdk/ddf-sso-widget/src/main/resources/OSGI-INF/blueprint/casfilters.xml]: Not if I remove the cas client dependency. I'll remove it as well.

File: features/security/pom.xml [hunk 2]
```diff
@@ -121,26 +121,6 @@
             <artifactId>security-expansion-commands</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
```
Comment (SmithJosh) [inline features/security/pom.xml]: This pom still references the `security-sts-casclaimshandler` dependency

Comment (blen-desta) [reply features/security/pom.xml]: Great catch.

**Comment by blen-desta** [comment]
build now

**Comment by bakejeyner** [comment]
General DDF smoke test hero successful ✔️

**Pull request merged**